src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -586,10 +588,15 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
 
     @wire(getRecord, {
         recordId: '$recordId',
-        fields: [BATCH_NAME, EXPECTED_COUNT_OF_GIFTS,
+        fields: [BATCH_ID_FIELD,
```

Comment:
Could we also add `BATCH_ID_FIELD` to a new line like entries in `optionalFields` below or otherwise one line this array?

Reply:
👍🏽